### PR TITLE
PL - Wanted

### PR DESCRIPTION
**Pull request by codecleaner**

### CODE REVIEW

**Comment by codecleaner** [comment]
Could you tell me when will it be available online?

**Comment by CodeClubRik** [comment]
@martinpeck is codeclubrobot in a position to create a pull request for these new projects? If not, it's no problem for me to manually rebuild the site.

**Comment by codecleaner** [comment]
@CodeClubRik @martinpeck Could you please generate CC projects site? I will really appreciate it because we need it to run our Code Clubs in Poland.

**Comment by martinpeck** [comment]
We're having some build issues at present, but we'll get it built soon.

**Comment by codecleaner** [comment]
@martinpeck Could I help you in some way with those issues?

**Comment by martinpeck** [comment]
Thanks, but it's all sorted. I should have said...sorry.

Your changes should be on http://projects.codeclubworld.org/pl-PL/05_html_01/index.html now.

**Comment by CodeClubRik** [comment]
@codecleaner I've noticed something strange happening with the trinket iframe for the 'wanted' project.
It seems to be showing the website's header and footer.

screen shot 2015-09-24 at 21 31 24

File: pl-PL/Wanted/Wanted.md [hunk 1]
```diff
@@ -9,24 +9,25 @@ stylesheet: web
...
-Let's start by editing the CSS code for the poster.
 
 ## Zadania do wykonania {.check}
 
-+ Open this trinket: <a href="http://jumpto.cc/web-wanted" target="_blank">jumpto.cc/web-wanted</a>. If you're reading this online, you can also use the embedded version of this trinket below.
++ Otwórz edytor: <a href="http://jumpto.cc/html-poszukiwany" target="_blank">jumpto.cc/html-poszukiwany</a>. Jeśli pracujesz online, możesz również posłużyć się wersją wyświetloną poniżej.
 
 <div class="trinket">
-	<iframe src="https://trinket.io/embed/html/58318bee1f" width="100%" height="550" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen>
+	<iframe src="https://trinket.io/html/aa8cd3c9f4" width="100%" height="550" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen>
```
Comment (martinpeck) [inline pl-PL/Wanted/Wanted.md]: this is missing the "embed" within the url

Comment (martinpeck) [inline pl-PL/Wanted/Wanted.md]: I've fixed it directly in our repository @CodeClubRik @codecleaner

**Comment by martinpeck** [comment]
This'll get picked up by the nightly build....unless you want to kick it off before then @CodeClubRik

**Comment by CodeClubRik** [comment]
Sorry, just seen this -- the site's been rebuilt now.

File: pl-PL/Wanted/Wanted.md (hunk 1 above)
Comment (codecleaner) [inline pl-PL/Wanted/Wanted.md]: @CodeClubRik @martinpeck Thank you both for your help with that!

**Comment by codecleaner** [comment]
@CodeClubRik @martinpeck Thanks for rebuild the site!

**Pull request merged**